modules/gateways/bitpaycheckout/bitpaycheckout_ipn.php

```diff
@@ -70,7 +73,7 @@ function checkInvoiceStatus($url){
 if($btn_id):
 switch ($event['status']) {
      #complete, update invoice table to Paid
-     case 'complete ':
+     case 'complete':
```

Comment:
Don't really know why that space was there but just to be sure you can add trim() to 
switch ($event['status']) just like switch ($event['status'].trim())
and test it